Nos/Views/Discover/FeaturedAuthorCategory.swift

```diff
@@ -8,27 +8,43 @@ enum FeaturedAuthorCategory: CaseIterable {
     case all
     /// A special type of category that includes all authors from the latest cohort.
     case new // swiftlint:disable:this identifier_name
-    case music
-    case news
-    case art // swiftlint:disable:this identifier_name
     case activists
-    case tech
-    case health
+    case art // swiftlint:disable:this identifier_name
+    case espanol
+    case food
     case gaming
+    case health
+    case lifestyle
+    case music
+    case news
+    case newzealand
+    case photography
+    case politics
+    case random
+    case scifi
     case sports
+    case tech
 
     var text: LocalizedStringResource {
         switch self {
         case .all: LocalizedStringResource.localizable.featuredAuthorCategoryAll
         case .new: LocalizedStringResource.localizable.featuredAuthorCategoryNew
-        case .music: LocalizedStringResource.localizable.featuredAuthorCategoryMusic
-        case .news: LocalizedStringResource.localizable.featuredAuthorCategoryNews
-        case .art: LocalizedStringResource.localizable.featuredAuthorCategoryArt
         case .activists: LocalizedStringResource.localizable.featuredAuthorCategoryActivists
+        case .art: LocalizedStringResource.localizable.featuredAuthorCategoryArt
```

Comment:
Would you mind converting this and the rest of them here to our new approach?
```suggestion
        case .art: "featuredAuthorCategoryArt"
```

Reply:
I didn't want to do this because Bryan has made those changes already.